### PR TITLE
CC-918: Update build to generate plugin package

### PR DESCRIPTION
**Pull request by wicknicks**
Signed-off-by: Arjun Satish <arjun@confluent.io>

### CODE REVIEW

File: kafka-connect-s3/pom.xml [hunk 1]
```diff
@@ -197,6 +197,53 @@
...
+                                        <pluginType>sink</pluginType>
+                                    </pluginTypes>
+
+                                    <tags>
+                                        <tag>s3</tag>
+                                        <tag>aws</tag>
+                                    </tags>
+
+                                    <requirements>
+                                        <requirement>Confluent Platform installed and running</requirement>
```
Comment (C0urante) [inline kafka-connect-s3/pom.xml]: We can probably leave this out as per @rhauch's comments on the other related PRs (https://github.com/confluentinc/kafka-connect-hdfs/pull/307 "Shouldn't a user be able to install this with just AK?")

File: kafka-connect-s3/pom.xml [hunk 2]
```diff
@@ -197,6 +197,53 @@
...
+
+                                        Being a sink, the S3 connector periodically polls data from Kafka and in turn uploads it to S3. A partitioner is used to split the data of every Kafka partition into chunks. Each chunk of data is represented as an S3 object, whose key name encodes the topic, the Kafka partition and the start offset of this data chunk. If no partitioner is specified in the configuration, the default partitioner which preserves Kafka partitioning is used. The size of each data chunk is determined by the number of records written to S3 and by schema compatibility.
+                                    </description>
+
+                                    <supportProviderName>Confluent, Inc.</supportProviderName>
+                                    <supportSummary>Confluent supports the S3 sink connector alongside community members as part of its Confluent Platform open source offering.</supportSummary>
+                                    <supportUrl>https://docs.confluent.io/current/</supportUrl>
+                                    <supportLogo>logos/confluent.png</supportLogo>
```
Comment (C0urante) [inline kafka-connect-s3/pom.xml]: Either this element should be removed for now, or a `logos/` directory should be created with the `confluent.png` image in it.

File: kafka-connect-s3/pom.xml [hunk 3]
```diff
@@ -197,6 +197,53 @@
                             </dependency>
                         </dependencies>
                     </plugin>
+                    <plugin>
+                        <groupId>io.confluent</groupId>
+                        <version>0.9.0</version>
```
Comment (C0urante) [inline kafka-connect-s3/pom.xml]: This version generates incorrectly-structured archive files, so we'll need to make sure to update this to either `0.9.1` or `1.0.0` (whichever is available at the time) before merging this PR.

File: kafka-connect-s3/pom.xml [hunk 4]
```diff
@@ -197,6 +197,53 @@
...
+                                    <supportProviderName>Confluent, Inc.</supportProviderName>
+                                    <supportSummary>Confluent supports the S3 sink connector alongside community members as part of its Confluent Platform open source offering.</supportSummary>
+                                    <supportUrl>https://docs.confluent.io/current/</supportUrl>
+                                    <supportLogo>logos/confluent.png</supportLogo>
+
+                                    <ownerUsername>confluentinc</ownerUsername>
+                                    <ownerType>organization</ownerType>
+                                    <ownerName>Confluent, Inc.</ownerName>
+                                    <ownerUrl>https://confluent.io/</ownerUrl>
+                                    <ownerLogo>logos/confluent.png</ownerLogo>
```
Comment (C0urante) [inline kafka-connect-s3/pom.xml]: See comment RE: `supportLogo`

File: kafka-connect-s3/pom.xml [hunk 5]
```diff
@@ -197,6 +197,53 @@
...
+                        <groupId>io.confluent</groupId>
+                        <version>0.9.0</version>
+                        <artifactId>kafka-connect-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>kafka-connect</goal>
+                                </goals>
+                                <configuration>
+                                    <title>Kafka Connect Storage Cloud</title>
```
Comment (C0urante) [inline kafka-connect-s3/pom.xml]: This should probably be `Kafka Connect S3`.

File: kafka-connect-s3/pom.xml (hunk 3 above)
Comment (wicknicks) [reply kafka-connect-s3/pom.xml]: Fixed other comments. Will update this once you push your changes. Thanks!

**Comment by C0urante** [comment]
@wicknicks as per offline discussion, can we set `<confluentControlCenterIntegration>` to `true`?

**Pull request merged**